src/CppAst.CodeGen/Common/CodeWriterOptions.cs

```diff
@@ -11,15 +11,11 @@ namespace CppAst.CodeGen.Common
 {
     public class CodeWriterOptions
     {
-        public CodeWriterOptions() : this(GetDefaultFileSystem(), CodeWriterMode.Full)
+        public CodeWriterOptions() : this(GetDefaultFileSystem())
         {
         }
 
-        public CodeWriterOptions(IFileSystem fileSystem) : this(fileSystem, CodeWriterMode.Full)
```

Comment:
I will let that change because CppAst.CodeGen is in a 0.x version, but keep in mind that this is considered as an API breaking change.

Reply:
My apologies, I'll be sure to keep and eye out document those more clearly.
I didn't mean to introduce any breaking changes in this type of pull request.